apps/projects/app/components/Panel/PanelComponents.js

```diff
@@ -40,14 +42,111 @@ export const IssueTitle = ({ issue }) => (
 )
 IssueTitle.propTypes = issueShape
 
-export const IssueText = styled.div`
-  white-space: nowrap;
-  overflow: hidden;
-  text-overflow: ellipsis;
+export const Avatar = ({ user }) => {
+  const theme = useTheme()
+
+  return (
+    <Link
+      href={user.url}
+      target="_blank"
+      style={{ textDecoration: 'none', color: `${theme.link}`, marginRight: 6 }}
+    >
+      <img
+        alt=""
```

Comment:
I'm curious about the impact to usability this had. We moved the link to the Avatar, rather than the username, and I think we are hiding the avatar from screen readers by setting `alt=""`. Maybe we could set the alt text to something like `{user.username} from GitHub`. Though that makes the text weird. "chadoh from GitHub chadoh applied 1 day ago" 😕

@javieralaves this might be a question for you. I actually think the way we had it before makes it easier to make the experience good for both sighted and blind users. We could easily hide the avatar from blind users, setting `alt=""`, and then make both the avatar and the username link to GitHub. Can we do that? What was the thinking behind removing the link from the username?

Reply:
@chadoh The objective was to focus the use of hyperlinks around applications and work submissions, but at the same time I don't have a strong opinion on bringing back the profile link as well. Note that as we move onto the 'GitHub decoupling' territory, some additional changes will be taking place very soon to this panel, in specific making use of some form of AddressBadge instead of what we have at the moment pointing at a GitHub profile; so we may want to wait until then to decide more in detail how to enhance accessibility.

Reply:
ok, I like keeping as-is for now. Let's think about this when we decouple from GitHub, as you suggest.